tests/sentry/api/test_event_search.py

```diff
@@ -964,12 +972,26 @@ def test_invalid_duration_filter(self):
             parse_search_query("transaction.duration:>..500s")
 
     def test_invalid_aggregate_duration_filter(self):
-        with self.assertRaises(InvalidSearchQuery, expected_regex="not a valid duration value"):
-            parse_search_query("avg(transaction.duration):>..500s")
+        assert parse_search_query("avg(transaction.duration):>..500s") == [
+            SearchFilter(
+                key=SearchKey(name="message"),
+                operator="=",
+                value=SearchValue(raw_value="avg(transaction.duration):>..500s"),
```

Comment:
The grammar no longer matches these at all since it's now constrained to real numbers.